r2ai/interpreter.py

```diff
@@ -23,12 +24,15 @@
 from signal import signal, SIGINT
 import sys
 
+ANSI_REGEX = re.compile(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])')
+
 file_dir = os.path.dirname(__file__)
 sys.path.append(file_dir)
 import index
 
 r2clippy = False
 have_rlang = False
+sysprint = print
```

Comment:
Could we have a `r2lang.write`? Or just keep using `sys.stdout.write`? It seems to not like it when I use `r2lang.print` and `sys.stdout.write` together.

Reply:
uhm.. but r2lang.X is buffering output into RCons, wont be shown until flushed to the console. its not suposed to be used as in live mode. If thats fine we can add it in the rlang api. otherwise i would suggest to buffer the output while printing it and then just do one call at the end? do this work for u?

you can also use the builtins.print. but that wont be captured by r2

Reply:
yeah makes sense, i dug into the r2lang code a bit and have a branch with a r2lang.write exposed but im not sure we even need r2 to be aware of the output of the LLM stream. Do you think we need to?
If you're ok with using the builtins.print or even just how i have it with stdout.write, I'll keep it like this.

Reply:
the only reason to have r2 aware of the output from r2ai is when its going to be used for scripting it via r2pipe/r2js, and i made r2ai.interactive to use builtins.print or r2lang.print depending on that variable just to let the user have an interactive output or a bfufered one via rcons.

so i think auto should just honor this behaviour. what do you think?